.gitignore

```diff
@@ -38,3 +38,6 @@ cypress/fixtures
 cypress/plugins
 cypress/support
 cypress/screenshots
+
+#docker db files
+/db/docker-data
```

Comment:
Looks like your edit isn't adding new lines automatically. In VSC you can do that with `"files.insertFinalNewline": true,` in your settings.json